scripts/release/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # that mimics Vagrant environment as far as required
 # for building the scripts and running provision scripts
 
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```

Comment:
Good call on keeping it here till we do more testing... some 20.04 setups have a lot of quirks with snaps that we might want to be aware of before we go charging in.